requirements.txt

```diff
@@ -20,3 +20,5 @@ pyasn1-modules>=0.2.1 # BSD
 jinja2>=3.0.0 # BSD
 # Keylime requires also gpg (LGPLv2+).
 # Those bindings must match the local GPG version and therefore this package is not installed via PyPI.
+# The typing-extensions module is currently required for Python versions < 3.8
```

Comment:
Does it mean it won't be needed for C9S with Python 3.9.14, just for C8S?

Reply:
It depends on what features one wants from it.Some of the newer features are only available in 3.10 I read.

Reply:
We are currently using Literal and TypedDict. If we use something else we can just bump the requirement.